cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000"
```

Comment:
will it run a local server while running the tests ? if so , before/after wds ?

I guess using [this](https://github.com/bahmutov/start-server-and-test) might be helpful in case of testing the app running in local using wds .

Reply:
Nope, we have to start webpack dev server before running cypress tests. But yes, if we consider integrating cypress into CI, we could automate it according to https://docs.cypress.io/guides/guides/continuous-integration.html#Boot-your-server